### DISCUSSION TITLE
Ability to adjust sync in replay mode / Video of Demand

### DISCUSSION THREAD

**Discussion by albertosottile** [f1multiviewer/issue-tracker#535] [Feature requests]
When replaying an event after sync data, the streams are supposed to be in sync with each other (thanks to the great manual work of MultiViewer). 

However, sometimes it happens to me that the sync is off (this happened e.g. at the latest USGS, where the on-boards were like 1 seconds behind w.r.t. the data tracker and the international feed. 

I do not understand why this happens but, for when it happens, it would be great to have the possibility to adjust the sync manually also when watching in Replay mode. Instead, when clicking on the Sync button, this UI appears 
![Screenshot 2024-10-24 at 23 32 07](https://github.com/user-attachments/assets/29829c7b-9400-43c8-80b8-1089131eec14)

and no adjustment is possible.

Would it be possible to implement the same delay/sync control also in Video on Demand mode?

**Comment by LapsTimeOFF** [comment]
We are going to work on making that eventually possible, in the meanwhile, when you spot issues like that, please report them on our [Discord](https://muvi.gg/discord) in a ticket (in #help) or in #sync
